File: JavaBasic/174_common_elements.java
// 174	174_common_elements.java	Find common elements between two arrays (intersection).
import java.util.*;
public class _174_common_elements {
    public static void main(String[] args) {
        int[] arr1={1,2,3,4,5}, arr2={3,4,5,6,7};
        List<Integer> common = new ArrayList<>();
        for (int x:arr1) for (int y:arr2) if(x==y&&!common.contains(x)) common.add(x);
        System.out.println("Common: " + common);
    }
}

File: JSBASIC/174_common_elements.js
// 174	174_common_elements.js	Find common elements between two arrays (intersection).

let arr1 = [1, 2, 3, 4, 5];
let arr2 = [3, 4, 5, 6, 7];
let common = arr1.filter(x => arr2.includes(x));
console.log("Array 1:", arr1);
console.log("Array 2:", arr2);
console.log("Common elements:", common);
